static_report/src/components/shared/Tables/TableList/CRTableListItem/index.tsx

```diff
@@ -35,90 +31,65 @@ export function CRTableListItem({
     baseTableDatum?.description || targetTableDatum?.description;
 
   return (
-    <AccordionButton
-      bgColor="white"
-      borderRadius="md"
-      data-cy="cr-table-overview-btn"
-    >
-      <Flex
-        direction="column"
-        gap={4}
-        py="10px"
-        maxH={isExpanded ? '135px' : '90px'}
+    <TableListItem isExpanded={isExpanded} data-cy="cr-table-overview-btn">
+      <Grid
+        templateColumns="218px 2fr 1.5fr 2.8rem"
+        justifyItems="flex-start"
+        width="calc(900px - 30px)"
       >
-        <Grid
-          templateColumns="218px 2fr 1.5fr 2.8rem"
-          justifyItems="flex-start"
-          width="calc(900px - 30px)"
-        >
-          <GridItem>
-            <Center>
-              <Icon as={FiGrid} color="piperider.500" />
-              <Text mx={1}>{columnName}</Text>
-
-              {!isExpanded && (
-                <Tooltip
-                  shouldWrapChildren
-                  placement="right-end"
-                  label={description}
-                >
-                  <Icon as={FiAlertCircle} ml={1} />
-                </Tooltip>
-              )}
-            </Center>
-          </GridItem>
-          <GridItem>
-            <Flex gap={10} color="gray.500">
-              <Text>Rows</Text>
-              <CRTableListDeltaSummary
-                baseCount={baseTableDatum?.row_count}
-                targetCount={targetTableDatum?.row_count}
-              />
+        <GridItem>
+          <TableItemName
+            name={columnName as string}
```

Comment:
suggest: handle `name` as `name?` in `TableItemName`

Reply:
Here i prefer to give it a default value.